generate_version_4a.py

```diff
@@ -0,0 +1,80 @@
+"""
+Generate a version 4a database file for qcodes' test suite to consume.
+
+Version 4a accidentally serialised the RunDescriber with an InterDependencies_
+object instead of an InterDependencies object
+"""
+
+import os
+import numpy as np
+
+# NB: it's important that we do not import anything from qcodes before we
+# do the git magic (which we do below), hence the relative import here
+import utils as utils
+
+
+def generate_DB_file_with_some_runs():
+    """
+    Generate a .db-file with a handful of runs with some interdependent
+    parameters
+    """
+
+    # This function will run often on CI and re-generate the .db-files
+    # That should ideally be a deterministic action
+    # (although this hopefully plays no role)
+    np.random.seed(0)
+
+    v3fixturepath = os.path.join(utils.fixturepath, 'version4a')
```

Comment:
should the variable be `v4afixturepath`?

Reply:
That would indeed be reasonable. Thanks.